.github/workflows/satellites.yml

```diff
@@ -0,0 +1,181 @@
+name: Test on Satellites
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore: [ docs/** ]
+  pull_request:
+    branches: [ main ]
+    paths-ignore: [ docs/** ]
+
+jobs:
+  tests:
+    name: +test +test-fail (satellites)
+    runs-on: ubuntu-latest
+    env:
+      FORCE_COLOR: 1
+      EARTHLY_TOKEN: "${{ secrets.EARTHLY_TOKEN }}"
+      EARTHLY_INSTALL_ID: "earthly-githubactions"
+      EARTHLY_ORG: "earthly-technologies"
+    steps:
+      - uses: earthly/actions-setup@v1.0.1
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+        with:
+          image: tonistiigi/binfmt:latest
+          platforms: all
+      - uses: actions/checkout@v3
+      - name: Docker mirror login (Earthly Only)
+        run: docker login registry-1.docker.io.mirror.corp.earthly.dev --username "${{ secrets.DOCKERHUB_MIRROR_USERNAME }}" --password "${{ secrets.DOCKERHUB_MIRROR_PASSWORD }}"
+        if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
+      - name: Configure Satellites
+        run: earthly satellite select earthly-core
```

Comment:
I suspect we'll want a `if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository` here to avoid this from running in forks?

Currently if this runs in a fork, that user shouldn't have access to `secrets.EARTHLY_TOKEN`, and then this will fail.

Reply:
FYI I tested this from a fork - these just run on a local buildkit instead and pass